services/QuillLMS/spec/controllers/admin_report_filter_selections_controller_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe AdminReportFilterSelectionsController, type: :controller do
+  let(:user) { create(:user) }
+  let(:valid_attributes) { { report: AdminReportFilterSelection::DATA_EXPORT, user_id: user.id } }
+  let(:invalid_attributes) { { report: 'invalid_report', user_id: user.id } }
```

Comment:
It feels a little odd to include `user_id` in these attribute variables given that you use them as part of request payloads (and `user_id` is not a valid param for the controllers.  I'm pretty sure you're doing this so that you can use the same value for both the request and for the `create` calls for model creation, so this isn't a huge deal, but I think I'd be inclined to break "params for HTTP requests" and "data for creating backend models" into two separate values if only to avoid confusion if someone reads this file and thinks they can pass arbitrary `user_id` values in.  Maybe something like:
```ruby
let(:valid_attributes) { { report: AdminReportFilterSelection::DATA_EXPORT, user_id: user.id } }
let(:invalid_attributes) { { report: 'invalid_report' } }
let(:valid_model_attributes { valid_attributes.merge({ user_id: user.id }) }
```